python/seldon_core/seldon_client.py

```diff
@@ -1478,19 +1478,19 @@ def rest_predict_gateway(
             request = prediction_pb2.SeldonMessage(data=datadef, meta=metaKV)
         payload = seldon_message_to_json(request)
 
-    if not headers is None:
+    if headers is not None:
```

Comment:
Some drive-by syntax fixes, there are plenty more but didn't want to go down a rabbit hole.